app/views/organizations/staff/dashboard/index.html.erb

```diff
@@ -1,75 +1,109 @@
               </div>
-            </div>
+            <% end %>
           <% end %>
-        <% end %>
+        </div>
       </div>
+
       <div class="col-xl-3 col-lg-6 col-md-12 col-12">
-        <%= render DataCardComponent.new do |card| %>
-          <% card.with_title { t(:fosters) } %>
-          <% card.with_icon_name { "heart" } %>
-          <% card.with_value do %>
-            <div>
-                <%= link_to "#{@current_foster_count} Current",
-                staff_manage_fosters_path(q: { status_eq: "current" })
-                %>
-            </div>
-            <div>
-                <%= link_to "#{@upcoming_foster_count} Upcoming",
-                staff_manage_fosters_path(q: { status_eq: "upcoming" })
-                %>
-            </div>
+        <div class="foster-card">
+          <%= render DataCardComponent.new(title_size: :medium, icon_color: "text-warning") do |card| %>
+            <% card.with_title { t(:fosters) } %>
+            <% card.with_icon_name { "heart" } %>
+            <% card.with_value do %>
+              <div class="fs-4 ms-1">
+                <div>
+                  <%= link_to staff_manage_fosters_path(q: { status_eq: "current" }), 
+                      class: "text-decoration-none text-gray-500 fw-normal" do %>
+                    <i class="fe fe-calendar text-warning me-1"></i>
+                    <%= "#{@current_foster_count} Current" %>
+                  <% end %>
+                </div>
+                <div>
+                  <%= link_to staff_manage_fosters_path(q: { status_eq: "upcoming" }), 
+                      class: "text-decoration-none text-gray-500 fw-normal" do %>
+                    <i class="fe fe-clock text-warning me-1"></i>
+                    <%= "#{@upcoming_foster_count} Upcoming" %>
+                  <% end %>
+                </div>
+              </div>
+            <% end %>
           <% end %>
-        <% end %>
+        </div>
       </div>
+
       <div class="col-xl-3 col-lg-6 col-md-12 col-12">
-        <%= render DataCardComponent.new do |card| %>
-          <% card.with_title { t(:rescues) } %>
-          <% card.with_icon_name { "star" } %>
-          <% card.with_value { "TBD" } %>
-        <% end %>
+        <div class="rescue-card">
+          <%= render DataCardComponent.new(title_size: :medium, icon_color: "text-gray-500") do |card| %>
+            <% card.with_title { t(:rescues) } %>
+            <% card.with_icon_name { "star" } %>
+            <% card.with_value do %>
+              <div class="text-muted text-decoration-none text-gray-500 fw-normal">
+                <div class="fs-4 ms-1">
+                  TBD
+                </div>
+              </div>
+            <% end %>
+          <% end %>
+        </div>
       </div>
     </div>
-    <%= render 'pets_with_incomplete_or_overdue_tasks' %>
+
+    <div class="mt-5">
+      <%= render 'pets_with_incomplete_or_overdue_tasks' %>
+    </div>
   <% end %>
-<% end %>
+<% end %>
```

Comment:
New line :)

Reply:
Done. I think need to add some ERB Lint check as well to avoid such issue.